app/javascript/campaigner_facing/page_edit_bar.js

```diff
@@ -1,8 +1,12 @@
-import ErrorDisplay from "../shared/show_errors";
-import _ from "lodash";
+import $ from 'jquery';
+import I18n from 'champaign-i18n';
+import Backbone from 'backbone';
```

Comment:
How come it worked without all these imports?

Reply:
We had `window.$` loaded via CDN - but we're getting a lot of errors on sentry on jQuery not being present, which was the main reason for this change in how we load jQuery.